### PR TITLE
Show help tooltip on hover in sizing tool.

### LINKED ISSUE #7889: [Sizing Tool] Show help text on hover
The sizing tool has a ? icon next to each field label. On hover this should show a help text.

### PR DESCRIPTION
**Pull request by jeschkies**
**What this PR does / why we need it**:
The labels for the sizing tool require some additional description. This change introduces help text on hover.

**Which issue(s) this PR fixes**:
Fixes #7889

**Special notes for your reviewer**:

**Checklist**
- [ ] Reviewed the `CONTRIBUTING.md` guide
- [x] Documentation added
- [ ] Tests updated
- [ ] `CHANGELOG.md` updated
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/upgrading/_index.md`

### CODE REVIEW

File: docs/sources/installation/sizing/index.md [hunk 1]
```diff
@@ -42,6 +42,21 @@ This tool helps to generate a Helm Charts `values.yaml` file based on specified
...
+    Defines the log volume in terrabytes expected to be ingested each day.
+    </span>
+    <span v-else-if="help === 'node'">
+    Defines the node type of the Kubernetes cluster.
+    </span>
+    <span v-else-if="help === 'retention'">
+    Defines how long the ingested logs should be kept.
+    </span>
+    <span v-else-if="help === 'queryperf'">
+    Defines the expected query performance. Basic is... Super is...
```
Comment (JStickler) [inline docs/sources/installation/sizing/index.md:57]: "Basic is... Super is..."  Was there going to be more text here?

Comment (jeschkies) [reply docs/sources/installation/sizing/index.md:57]: Yes. I need @trevorwhitney to explain when to use Basic and when to use Super.

Comment (trevorwhitney) [reply docs/sources/installation/sizing/index.md:57]: Basic is ~3mbps, Super is better than that (I don't have exact numbers, we would need to run tests to get them)

**Review by trevorwhitney — APPROVED** [review]
lgtm, thanks!

File: docs/sources/installation/sizing/index.md (hunk 1 above)
Comment (jeschkies) [reply docs/sources/installation/sizing/index.md]: @JStickler could you take another look :slightly_smiling_face:

**Pull request merged**